File: ariarh-middleoffice/src/main/metamodel/com/itpeac/ariarh/middleoffice/domain/Category_.java
package com.itpeac.ariarh.middleoffice.domain;

import javax.annotation.Generated;
import javax.persistence.metamodel.ListAttribute;
import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;

@Generated(value = "org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor")
@StaticMetamodel(Category.class)
public abstract class Category_ extends com.itpeac.ariarh.middleoffice.domain.AbstractAuditingEntity_ {

	public static volatile SingularAttribute<Category, Long> id;
	public static volatile SingularAttribute<Category, String> label;
	public static volatile ListAttribute<Category, Question> questionList;

	public static final String ID = "id";
	public static final String LABEL = "label";
	public static final String QUESTION_LIST = "questionList";

}

File: ariarh-webapp/src/main/web/src/app/demo/api/category.ts
import {Question} from "./question";

export class Category{
    _id?:number;
    label?:string;
    questionList?:Question[]

    constructor(id?:number,label?:string, questionList?:Question[]) {
        this._id = id
        this.label = label
        this.questionList = questionList

    }

    get id():number{
        return this.id
    }
}
